### PR TITLE
feat: Flag expiry

### PR DESCRIPTION
**Pull request by aronshamash**
- As per https://nidigitalsolutions.jira.com/browse/REPLAT-5484
- `flags` field on `Article` is now deprecated. This PR uses `expirableFlags`
- Also needed to update `types.ts` which hadn't been done in a while

### CODE REVIEW

File: packages/article-in-depth/src/article-header/article-header.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ const ArticleHeader = ({
               {headline}
             </Text>
 
-            {flags.length > 0 && (
+            {getActiveFlags(flags).length > 0 && (
```
Comment (nehasri89) [inline packages/article-in-depth/src/article-header/article-header.js]: ain't we doing this again in `article-flag`, think we can skip this call from here

File: packages/article/showcase-helper.js [hunk 2]
```diff
@@ -192,6 +192,9 @@ const renderArticle = ({
         template,
         textColour: inDepthTextColour
       };
+
+      console.log("data", data);
```
Comment (nehasri89) [inline packages/article/showcase-helper.js]: this can go

File: packages/article-flag/package.json [hunk 3]
```diff
@@ -58,7 +58,7 @@
     "react-native": "0.55.4",
     "react-native-web": "0.9.0",
     "react-test-renderer": "16.5.2",
-    "webpack": "4.6.0",
+    "webpack": "4.29.6",
```
Comment (nehasri89) [inline packages/article-flag/package.json]: why does it need a webpack upgrade?

Comment (aronshamash) [reply packages/article-flag/package.json]: good spot. not intentional

**Pull request merged**